QUICKSTART.md

```diff
@@ -12,14 +12,89 @@ Once you have a cluster, you can skip to the last step, [Launch!](#launch)
 ## Dependencies
 
 EKS has a command-line tool called `eksctl` that makes cluster setup easy.
-First, get [eksctl set up](https://github.com/weaveworks/eksctl).
+Versions of eksctl starting with 0.15.0-rc.2 support Bottlerocket natively.
+We recommend that you download the [latest version of eksctl](https://github.com/weaveworks/eksctl/releases) to get this support.
 
 You'll also need to [install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to augment `eksctl` during setup, and to run pods afterward.
 
 Finally, you'll need [aws-cli v1](https://aws.amazon.com/cli/) set up to interact with AWS.
 (You'll need a [recent v1 release](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html#install-tool-bundled) with EKS support.)
 
-## Finding an AMI
+## Automated setup
+
+If you have a recent `eksctl`, as mentioned above, most of Bottlerocket setup for EKS is automated.
+
+### Cluster setup
+
+#### Cluster setup configuration file
+
+eksctl can use a configuration file to simplify setup.
```

Comment:
Let's provide a link to `eksctl`'s config file documentation too: https://eksctl.io/usage/schema/

I think there's some other fields that folks will be interested in, so let's show them where to find them!